peeley/lucy
# Settings frontend

To test: 
1. Log in with an existing user** or create a new user, and navigate to /user-settings. 
2. Click on either button to view your audio or guided use settings.
3. Change the settings and click on the update button
4. view the settings that you changed to make sure it updated
5. make sure the "close settings" buttons work.

**users created before I made this branch may not have their own settings assigned to them, so it might be easier to create a new user. you can double check their settings in tinker.
